spec/factories/users.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :user do
-    client_slug { ENV['CLIENT_SLUG_DEFAULT'] || 'ncr' }
+    client_slug { ENV['CLIENT_SLUG_DEFAULT'] || nil }
```

Comment:
oh really? I grepped for it and only saw it referenced in tests. 

Up to you if we should remove - how much more cleanup would it require?